src/server/common/constants/scopes.js

```diff
@@ -2,7 +2,8 @@ const scopes = {
   admin: 'admin',
   tenant: 'tenant',
   serviceOwner: 'serviceOwner',
-  externalTest: 'externalTest'
+  externalTest: 'externalTest',
+  breakglass: 'breakglass'
```

Comment:
is breakglass 1 word? I'd be tempted to camelCase this `breakGlass`

Reply:
Can do. Tho it's breakglass everywhere now. I'll leave it as this. Let's discuss and if needed I'll raise PRs to change it all.